src/app/view/modules/form/project-form/project-form.component.ts

```diff
@@ -178,8 +195,27 @@ export class ProjectFormComponent implements OnInit {
         }
 
         this.form = this._fb.group({
-            'shortname': new FormControl({value: proj.shortname, disabled: this.iri !== undefined}, [Validators.required, Validators.minLength(3), Validators.maxLength(16), existingNamesValidator(this.existingShortNames), Validators.pattern(this.shortnameRegexp)]),
+
+            'shortname': new FormControl({
+                value: proj.shortname, disabled: this.iri !== undefined
+            }, [
+                Validators.required,
+                Validators.minLength(3),
+                Validators.maxLength(16),
```

Comment:
Maybe you could add something like `shortnameMinLength` and `shortnameMaxLength` (like you have done for shortcode) and use it here instead of numbers.

Reply:
Oh yes, I made this already, but It's now in another branch. I will correct it...